lib/matplotlib/dviread.py

```diff
@@ -927,26 +981,27 @@ def _parse(self, file):
 
         state = 0
         for line in file:
-            comment_start = line.find('%')
+            line = six.b(line)
+            comment_start = line.find(b'%')
             if comment_start > -1:
                 line = line[:comment_start]
             line = line.strip()
 
             if state == 0:
```

Comment:
I tried googling without much success the format of enc files but it looks like this is looking for patterns for the form
```
/FooEncoding [ /abc/def/ghi ] def
```
and returning the slash-separated words within the brackets.  At least the searching for brackets-surrounded fragments can clearly be rewritten using re.findall as above.